projects/kit/components/like/like.style.less

```diff
@@ -0,0 +1,88 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+@keyframes tuiLikeScale {
+    0% {
+        transform: scale(1);
+    }
+
+    50% {
+        transform: scale(1.4);
+    }
+
+    100% {
+        transform: scale(1.1);
+    }
+}
+
+/**
+ * @name Like
+ * @selector [tuiLike]
+ *
+ * @description
+ * A stylized input type="checkbox"
+ *
+ * @attributes
+ * data-size — size (default: 'm') ('s' | 'm')
+ *
+ * @vars
+ * --t-icon-color - custom color for `:checked` state
+ * --t-unchecked-icon — default state icon
+ * --t-checked-icon — checkmark icon
+ *
+ * @example
+ * <input tuiLike />
+ * <input tuiLike="var(--tui-status-negative)" />
+ *
+ * @see-also
+ * Checkbox, Switch, Radio, Appearance
+ */
+[tuiLike] {
+    --t-size: var(--tui-height-m);
+    --t-border-width: 0.75rem;
+
+    inline-size: var(--t-size);
+    block-size: var(--t-size);
+    color: var(--tui-text-action);
+    background-color: var(--tui-background-neutral-2);
+    border: var(--t-border-width) transparent solid;
+    border-radius: 100%;
+    cursor: pointer;
+    margin: 0;
+    flex-shrink: 0;
+
+    &::before,
+    &::after {
+        .fullsize();
+        .transition(~'transform, mask, opacity, color');
+
+        content: '';
+        display: block;
+        background: currentColor;
+        mask: var(--t-unchecked-icon) no-repeat center/contain;
+    }
+
+    &::after {
+        mask-image: var(--t-checked-icon);
+        opacity: 0;
+    }
+
+    &:checked {
+        &::before {
+            opacity: 0;
+        }
+
+        &::after {
+            opacity: 1;
+            animation: tuiLikeScale var(--tui-duration) forwards;
+        }
+
+        &:not([tuiLike=''])::after {
```

Comment:
I'm not sure this attribute is still there in prod build, that's why we have `:host { tuiButton: '' }` in all such cases. Maybe just `color: var(--t-icon-color, inherit)`?

Reply:
`color: var(--t-icon-color, inherit)` unfortunately not working due to css priority.

<img width="450" alt="image" src="https://github.com/user-attachments/assets/c9244032-c199-4a6a-ac0d-2b973100ed06">


 It seems like, we don't attach `color` to prevent such behaviour or use current solution with `&:not([tuiLike=''])`. Are there another options?